### PR TITLE
Add option to export Excel file

### PR DESCRIPTION
**Pull request by J535D165**
This PR adds an option to export results to EXCEL (xlsx). Defaults to Excel.

localhost_3000_(iPad)
localhost_3000_(iPad) (1)

### CODE REVIEW

**Comment by J535D165** [comment]
@qubixes, can you do a review of the Python code. It required more lines of code than I expected :|

File: asreview/data.py [hunk 1]
```diff
@@ -556,8 +556,48 @@ def to_dataframe(self, labels=None, ranking=None):
...
+        """Export to csv.
+
+        Arguments
+        ---------
+        fp: str, NoneType
+            Filepath or None for buffer.
+        labels: list, np.ndarray
+            Current labels will be overwritten by these labels
+            (including unlabelled). No effect if labels is None.
+        df_order: list
```
Comment (qubixes) [inline asreview/data.py:568]: df_order -> ranking

File: asreview/data.py [hunk 2]
```diff
@@ -556,8 +556,48 @@ def to_dataframe(self, labels=None, ranking=None):
...
+            Reorder the dataframe according to these (internal) indices.
+            Deafault ordering if df_order is None.
```
Comment (qubixes) [inline asreview/data.py:570]: Deafault -> Default, df_order -> ranking

File: asreview/data.py [hunk 3]
```diff
@@ -556,8 +556,48 @@ def to_dataframe(self, labels=None, ranking=None):
...
```
Comment (qubixes) [inline asreview/data.py:592]: Deafault -> default, df_order -> ranking

File: asreview/webapp/utils/paths.py [hunk 4]
```diff
@@ -55,6 +55,18 @@ def get_project_file_path(project_id):
     return Path(get_project_path(project_id), "project.json")
 
 
+def get_tmp_path(project_id):
```
Comment (qubixes) [inline asreview/webapp/utils/paths.py:58]: Perhaps we can make use of tempfile? https://docs.python.org/3.8/library/tempfile.html (tempfile.mkstemp)

File: asreview/webapp/src/Components/ExportDialog.js [hunk 5]
```diff
@@ -65,17 +92,32 @@ const ExportDialog = (props) => {
         <DialogTitle id="scroll-dialog-title">Export results {props.project_id}</DialogTitle>
           <DialogContent dividers={true}>
             <Typography>
-              Download the result or your review
+              Download the result or your review to a Excel or CSV file.
```
Comment (qubixes) [inline asreview/webapp/src/Components/ExportDialog.js:95]: a Excel -> an Excel

Maybe "Export your review (Excel or CSV)."?

File: asreview/webapp/utils/paths.py (hunk 4 above)
Comment (J535D165) [reply asreview/webapp/utils/paths.py:58]: Good suggestion, I will implement this. Maybe this is a good way to export the CSV as well. Easier to trace than exporting a buffer.

File: asreview/webapp/src/Components/ExportDialog.js (hunk 5 above)
Comment (J535D165) [reply asreview/webapp/src/Components/ExportDialog.js:95]: Thanks

**Comment by J535D165** [comment]
Thanks, made some changes. 

The config file suggestion is nice. Especially if there are also human-readable labels available to use in the front-end. Let's put this on our todo list. I added the option to `to_file`.

**Pull request merged**